packages/wallet/backend/src/incomingPayment/validation.ts

```diff
@@ -1,11 +1,24 @@
 import { z } from 'zod'
 
 export const incomingPaymentSchema = z.object({
-  body: z.object({
-    paymentPointerId: z.string().uuid(),
-    amount: z.number().positive(),
-    description: z.string().optional()
-  })
+  body: z
+    .object({
+      paymentPointerId: z.string().uuid(),
+      amount: z.number().positive(),
+      description: z.string().optional(),
+      expiry: z.coerce.number().positive().int().optional(),
```

Comment:
I suggest to have an optional object for expiration, and the value and unit to be required inside the object